pom.xml

```diff
@@ -534,6 +535,12 @@
         <version>${undertow.version}</version>
       </dependency>
 
+      <dependency>
```

Comment:
We don't need this if we upgrade to Spring Boot 2.1.13. It has the CVE fix already.

Reply:
ah!!! ok ok

Reply:
I thought we're staying on 2.1.12, then we would have had to override the reactor-netty.